app/routes/index.js

```diff
@@ -1,5 +1,52 @@
-module.exports = function(app) {
+var validUrl = require('valid-url');
+var normalizeUrl = require('normalize-url');
+var originalURLHandler = require('../api/originalURLHandler.js');
+var shortURLHandler = require('../api/shortURLHandler.js');
+
+var shortURLRegExp = new RegExp('^[0-9]{7}$', 'g');
+
+module.exports = function(app, dbURL) {
 	app.get('/', function(req,res) {
 		res.render('index');
 	});
+
+	app.get('/*', function(req,res) {
```

Comment:
nice! Moved the decision making to the router. Cleaning up params before forwarding them off to handlers. Did you at first try to put the regex into the first param of `app.get` and create 2 routes? It looks like that might be difficult, but it could simplify the logic in here.